objectivec/src/ort_coreml_execution_provider.mm

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "ort_coreml_execution_provider.h"
+
+#if __has_include("coreml_provider_factory.h")
```

Comment:
Can we use #ifdef USE_COREML here?

Reply:
I don't think so - in the pod, this will build separately from ORT